src/tests/Microsoft.PowerFx.Core.Tests/LexerTests.cs

```diff
@@ -260,24 +254,11 @@ public void TestLexGetFragment(string script, int min, int lim, string fragment)
             Assert.Equal(fragment, span.GetFragment(script));
         }
 
-        private ILanguageSettings SomeFrenchLikeSettings()
+        private CultureInfo SomeFrenchLikeSettings()
```

Comment:
>SomeFrenchLikeSettings [](http://example.com/codeflow?start=28&length=22)

[minor] Can you rename it to `GetFrenchSettings` now, since it's not just "French-like"? :-)